### PR TITLE
Add some fips endpoints. Perhaps a fix for the partition regex.

### PR DESCRIPTION
**Pull request by mmzx**
On _main_
`ExAws.Config.Defaults.host("sts", "us-east-1-fips")` would return `nil`.
With this PR, it returns
`"sts-fips.us-east-1.amazonaws.com"`

Plus,

Some endpoints are added.

### CODE REVIEW

**Comment by mmzx** [comment]
On _main_
`ExAws.Config.Defaults.host("sts", "us-east-1-fips")` would return `nil`.
With this PR, it returns
`"sts-fips.us-east-1.amazonaws.com"`

I will put the PR out of draft state when I review the urls again.

File: priv/endpoints.exs [hunk 1]
```diff
@@ -1036,6 +1040,11 @@
             "eu-north-1" => %{},
             "sa-east-1" => %{},
             "us-east-1" => %{"sslCommonName" => "{service}.{dnsSuffix}"},
+            "us-east-1-fips" => %{
+              "sslCommonName" => "{service}.{dnsSuffix}",
```
Comment (mmzx) [inline priv/endpoints.exs:1044]: I am not sure about the `sslCommonName` field, please do have it reviewed!

**Comment by mmzx** [comment]
ping @bernardd

**Comment by bernardd** [comment]
Thanks @mmzx!

**Pull request merged**